src/navigate-pop/launching-applications.md

```diff
@@ -0,0 +1,78 @@
+<!--Sources
+Pop Training Guide (Google doc)
+https://support.system76.com/articles/pop-basics/
+
+-->
+# Locate, Launch, and Navigate Between Applications
+
+---
+
+## Finding Installed Applications
+
+Installed applications can be viewed in the Applications menu and the Pop!\_Shop Installed screen.
+
+### The Applications Menu
+
+The Application menu can be accessed from the Dock or by clicking `Applications` in the upper left corner of the screen.
```

Comment:
Why is it "Application menu" instead of "Applications menu" just the one time?

Reply:
Good catch, fixed.